cluster/addons/dns/skydns-rc.yaml.in

```diff
@@ -35,7 +35,7 @@ desiredState:
                     "-domain={{ pillar['dns_domain'] }}",
             ]
           - name: skydns
-            image: kubernetes/skydns:2014-12-23-001
+            image: skynetservices/skydns:latest
```

Comment:
Agreed, but I didn't see any tagged versions in the SkyDns  docker repo and I wasn't aware of the size issue. Happy to update this if someone creates an updated, tagged image.